code/modules/reagents/chemistry/reagents/toxins.dm

```diff
@@ -502,6 +502,19 @@
 	M.adjustToxLoss(1)
 	..()
 
+/datum/reagent/somnistimine
+	name = "Somnistimine"
+	id = "somnistimine"
+	description = "A powerful tranquilizer."
+	reagent_state = LIQUID
+	color = "#2E2E61"
+	metabolization_rate = 0.4
+
+/datum/reagent/somnistimine/on_mob_life(mob/living/M)
+	if(current_cycle>= 3)
```

Comment:
Its not as split as you think. Atmos cannot go into engie areas. Engie cannot go into atmos. Mechnic makes pods. They are all collectively still a unit.

Last i knew: Robotcists could not go into chemlab/xenobio area. Science cannot go into the roboticist area. I think they all might work as a collective unit. Genecists are a Chimera.